components/automate-cds/server/server.go

```diff
@@ -0,0 +1,61 @@
+package server
+
+import (
+	"context"
+	"net"
+
+	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/reflection"
+	"google.golang.org/grpc/status"
+
+	grpc_s "github.com/chef/automate/api/interservice/cds/service"
+	"github.com/chef/automate/components/automate-cds/service"
+	"github.com/chef/automate/lib/grpc/health"
+	"github.com/chef/automate/lib/tracing"
+)
+
+// NewGRPCServer creates a grpc server that serves all Teams GRPC APIs
+func NewGRPCServer(s *service.Service) *grpc.Server {
+	g := s.ConnFactory.NewServer(
+		grpc.UnaryInterceptor(
+			grpc_middleware.ChainUnaryServer(
+				tracing.ServerInterceptor(tracing.GlobalTracer()),
+				InputValidationInterceptor(),
```

Comment:
As of now, there's no `Validate()` methods on this service's gRPC input messages, so this won't do a thing.

Maybe you're planning on including them, so this could stay. Just nitpicking.

Reply:
Thank you. I will look into this.